btap/src/predict.py

```diff
@@ -477,21 +484,23 @@ def fit_evaluate(args):
     data_json = json.dumps(results_pred).encode('utf-8')
 
     # copy data to minio
-    acm.access_minio(operation='copy',
+    write_to_minio = acm.access_minio(operation='copy',
                      path=args.output_path,
                      data=data_json)
-
+    
+    logger.info("write to mino  ", write_to_minio)
```

Comment:
Missing data placeholder in log statement.

```suggestion
    logger.info("write to mino %s", write_to_minio)
```